packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -171,43 +93,89 @@ export const HvTableRow = forwardRef<HTMLElement, HvTableRowProps>(
 
     const TableRow = useMemo(() => StyledTableRow(Component), [Component]);
 
+    let even = getVarValue(theme.table.rowStripedBackgroundColorEven);
+
+    let odd = getVarValue(theme.table.rowStripedBackgroundColorOdd);
+
+    let stripedColorEven = checkValidHexColorValue(even)
+      ? hexToRgbA(even, 0.6)
+      : even;
+
+    let stripedColorOdd = checkValidHexColorValue(odd)
+      ? hexToRgbA(odd, 0.6)
+      : odd;
+
+    useEffect(() => {
+      even = getVarValue(theme.table.rowStripedBackgroundColorEven);
+
+      odd = getVarValue(theme.table.rowStripedBackgroundColorOdd);
+
+      stripedColorEven = checkValidHexColorValue(even)
+        ? hexToRgbA(even, 0.6)
+        : even;
+
+      stripedColorOdd = checkValidHexColorValue(odd)
+        ? hexToRgbA(odd, 0.6)
+        : odd;
+    }, [activeTheme?.colors?.modes[selectedMode]]);
```

Comment:
Mutating variables like this can lead to all sorts of problems. We should use `useState`.
Also, `getVarValue` still gets run every render like this.

```suggestion
    const [even, setEven] = useState(getVarValue(theme.table.rowStripedBackgroundColorEven));
    const [odd, setOdd] = useState(getVarValue(theme.table.rowStripedBackgroundColorOdd));

    const stripedColorEven = checkValidHexColorValue(even)
      ? hexToRgbA(even, 0.6)
      : even;

    const stripedColorOdd = checkValidHexColorValue(odd)
      ? hexToRgbA(odd, 0.6)
      : odd;

    useEffect(() => {
      setEven(getVarValue(theme.table.rowStripedBackgroundColorEven));
      setOdd(getVarValue(theme.table.rowStripedBackgroundColorOdd));
    }, [activeTheme?.colors?.modes[selectedMode]]);
```